src/test/java/com/hydratereminder/command/CommandInvokerTest.java

```diff
@@ -85,6 +84,7 @@ public void shouldReturnNothingWhenIsNotHydrateCommand() {
 
         // when and then
         commandInvoker.invokeCommand(commandToExecute);
+        verify(commandCreator,never()).createFrom(any()); // used verify to assert 
```

Comment:
No comment needed here :)

Reply:
@AbbasSalloum please remove the comment here as mentioned above and add a space after the comma in the call to `verify(...)` call.